game/plugins/src/main/kotlin/gg/rsmod/plugins/content/cmd/addspawn.plugin.kts

```diff
@@ -39,14 +46,24 @@ fun tryWithUsage(player: Player, args: Array<String>, failMessage: String, tryUn
     }
 }
 
+/**
+ * Adds the spawn to the proper region file corresponding to the players location.
+ */
 fun addSpawn(player: Player, id: Int) {
-    val string = "spawn_npc(npc = Npcs.${getName(id)}, x = ${player.tile.x}, z = ${player.tile.z}, walkRadius = 5, direction = Direction.${player.faceDirection})"
+    if (getName(id) == null)
+        player.message("ERROR: val not found corresponding with given id; reverting to using supplied id instead.")
+    val string = "spawn_npc(npc = ${if (getName(id) == null) id else "Npcs.${getName(id)}"}, x = ${player.tile.x}, z = ${player.tile.z}, walkRadius = 5, direction = Direction.${player.faceDirection})"
     try {
         val file = File("./game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/spawns/spawns_${player.tile.regionId}.plugin.kts")
         if (!file.exists())
             createSpawnFile(file)
         val writer = BufferedWriter(FileWriter(file, true))
-        writer.write("$string //Added by command by user: ${Misc.formatForDisplay(player.username)}")
+
+        val lines = Files.readAllLines(Paths.get(file.path))
+        if (lines.last().isNotEmpty() || lines.last().isNotBlank())
```

Comment:
Please use brackets here :)

(Okay, we should be never be doing one line `if() ->` without brackets, as that is the proper convention for the source code.)